### DISCUSSION TITLE
How to load data in my component from diffent route?

### DISCUSSION THREAD

**Discussion by devbapidey** [inertiajs/inertia#844] [Help]
I have a form where I what to auto complete function. Autocomplete came from a different route. How to inject that data in my vue component?

**Comment by reinink** [comment]
For these kinds of things, I recommend just using plain XHR/fetch with an endpoint that returns JSON.

Not all requests/responses in an Inertia app *need* to be done via Inertia. Sometimes it makes sense to still use plain XHR/fetch. 👍
